mock/py/mockbuild/util.py

```diff
@@ -1698,7 +1700,7 @@ def add_local_repo(config_opts, baseurl, repoid=None, bootstrap=None):
 best=1
 """.format(repoid=repoid, baseurl=baseurl)
 
-    config_opts['yum.conf'] += localyumrepo
+    config_opts['{0}.conf'.format(config_opts['package_manager'])] += localyumrepo
```

Comment:
here , it beaks configuration if we want use dnf when we have yum.conf , for example epel7 , I guess

Reply:
Indeed.  Can you please consider #486, would that help?

Reply:
That said, it should work just fine if `package_manager` is in sync with `{yum,conf}.conf`.  It should
be the case in the latest mock-core-configs.

Reply:
why not revert these commits ? yum.conf for all ! :)

Reply:
what are the advantages, perspectives of having dnf.conf ?